packages/die.vm/die.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>die.vm</id>
-    <version>3.10.20250219</version>
+    <version>3.10.20250505</version>
     <authors>Hellsp@wn, horsicq</authors>
-    <description>Detect It Easy, or abbreviated "DIE" is a program for determining types of files.</description>
+    <description>DIE, Detect It Easy, is a program for determining types of files.</description>
     <dependencies>
       <dependency id="common.vm" version="0.0.0.20250206" />
     </dependencies>
     <tags>File Information</tags>
+    <projectUrl>https://github.com/horsicq/DIE-engine</projectUrl>
```

I think the GH project we get the installation binary from is not the tool home page:
```suggestion
    <projectUrl>https://github.com/horsicq/Detect-It-Easy</projectUrl>
```